Jenkinsfile

```diff
@@ -66,7 +66,7 @@ node(platform) {
 
     stage("Archive") {
         if (platform == 'android') {
-            archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-${BUILD_CONFIG}.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
```

Comment:
ok thanks for clarification. the change looks good. Is it deployed somewhere to test?